feature/gribi/otg_tests/base_hierarchical_nhg_update/base_hierarchical_nhg_update_test.go

```diff
@@ -138,222 +192,137 @@ func TestBaseHierarchicalNHGUpdate(t *testing.T) {
 	ate := ondatra.ATE(t, "ate")
 	top := configureATE(t, ate)
 
-	p2flow := "Port 1 to Port 2"
-	p3flow := "Port 1 to Port 3"
-	lbFlow := "Port 1 to Port 2 and Port 3"
-	createFlow(t, p2flow, top, &atePort2)
-	createFlow(t, p3flow, top, &atePort3)
-	createFlow(t, lbFlow, top, &atePort2, &atePort3)
-
 	ate.OTG().PushConfig(t, top)
 	ate.OTG().StartProtocols(t)
 
-	gribic, err := gribiClient(ctx, t, dut)
-	if err != nil {
-		t.Fatalf("Got error during gribi client setup: %v", err)
+	test := []struct {
```

Comment:
Please name it "tests" instead of "test" to indicate there are multiple tests.

Reply:
Done